osclib/conf.py

```diff
@@ -107,13 +107,35 @@
         'mail-noreply': 'noreply@opensuse.org',
         'mail-release-list': 'opensuse-releaseteam@opensuse.org',
     },
+    r'openSUSE:(?P<project>Backports:(?P<version>[^:]+))': {
+        'staging': 'openSUSE:%(project)s:Staging',
+        'staging-group': 'factory-staging',
+        'staging-archs': 'x86_64',
+        'lock': 'openSUSE:%(project)s:Staging',
+        'lock-ns': 'openSUSE',
```

Comment:
Perhaps add:

```python
r'openSUSE:(?P<project>(?:Backports|Factory|Leap)(?::.*)?)'
```

With all the common properties for all `openSUSE` products with a priority of `1`.

Reply:
Can decide if you want it to match sub projects as currently all patterns do even without the optional `(?::.*)?` on the end. As such you can drop that. `re.match()` is basically `re.startswith()`. Alternatively the patterns could be restricted by placing `$` on the end.

Reply:
there is a break in the loop, so only one setting gets applied atm